pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      String :email
+      Int :dashboard_user_id
+      String :name
+      String :street_address, size: 1024
+      String :city
+      String :state
+      String :country
+      String :postal_code
+      String :district_name
+      String :district_city
+      String :district_state
+      String :district_zip
+      String :school_name
+      String :roles
+      String :facilitator_type, size: 1024
```

Comment:
According to https://github.com/code-dot-org/code-dot-org/pull/11243, it looks like this will be used to store courses the teacher facilitates. So, it's actually a list. I'd suggest renaming to `courses_facilitated` or something similar. At least `facilitator_types` (plural).